### ISSUE TITLE
Error generating MLJ model registry

### ISSUE THREAD

**Issue by ablaom** [sylvaticus/BetaML.jl#37]
Running `MLJModels.@update` to update MLJ's model registry is running into this new error:

```julia
ERROR: LoadError: Bad `load_path` trait for BetaML.Imputation.BetaMLGMMImputer: BetaMLGMMImputer not a registered package. 
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] top-level scope
   @ ~/MLJ/MLJModels/src/registry/src/update.jl:122
 [3] eval
   @ ./boot.jl:373 [inlined]
 [4] eval(x::Expr)
   @ Base.MainInclude ./client.jl:453
 [5] _update(mod::Module, test_env_only::Bool)
   @ MLJModels.Registry ~/MLJ/MLJModels/src/registry/src/update.jl:153
 [6] var"@update"(__source__::LineNumberNode, __module__::Module)
   @ MLJModels.Registry ~/MLJ/MLJModels/src/registry/src/update.jl:24
in expression starting at REPL[4]:1
```

**Comment by sylvaticus** [comment]
Thanks, I'll solve this tomorrow.
Should I add `MLJModels.@update` to BetaML tests?

**Comment by ablaom** [comment]
No, but there is an *experimental* test suite now: https://github.com/JuliaAI/MLJTestIntegration.jl .

**Comment by sylvaticus** [comment]
Should have been solved [here](https://github.com/sylvaticus/BetaML.jl/commit/a4719c6b26c3f3c0f3e52c5a6334f617bdb875cc) (untested, on a mobile phone).
I need to solve something concerning the tree display and making new release tomorrow....

**Comment by sylvaticus** [comment]
Hello, just released v0.7.1 that should fix the problem. Please confirm, so I can close this issue.

**Comment by ablaom** [comment]
Confirmed.
